ci/run_grading.sh

```diff
@@ -1,4 +1,8 @@
 cd grading
 pip install -r requirements.txt
 
-python grading.py "../main" 2>&1 | stdbuf -oL tee feedback-log.txt
\ No newline at end of file
+python grading.py "../main" 1 2>&1 | stdbuf -oL tee feedback-ex1.txt
+python grading.py "../main" 2 2>&1 | stdbuf -oL tee feedback-ex2.txt
```

Comment:
I assume either this or the one in the README does not work, no? The code seems to read the exercise id from position 1 and the optional folder from position 2. So you'd need to call it the other way around, no?